.github/workflows/publish-release.yml

```diff
@@ -398,6 +398,65 @@ jobs:
         run: |
           echo "continue"
 
+  e2e-stateful-data-test:
+    needs:
+      - check_branch
+    runs-on: buildjet-4vcpu-ubuntu-2204
+    timeout-minutes: 120
+    steps:
+      - name: "Checkout Code"
+        if: ${{ github.event.inputs.skip_checks != 'true' }}
+        uses: actions/checkout@v3
+
+      - name: Execute stateful-data-test
+        if: ${{ github.event.inputs.skip_checks != 'true' }}
+        shell: bash
+        run: |
+          make start-e2e-import-mainnet-test
```

Comment:
can we put this in the advanced tests instead? I think it would be useful to run nightly on develop too